packages/bootstrap-4/src/RadioWidget/RadioWidget.tsx

```diff
@@ -22,45 +19,43 @@ const RadioWidget = ({
 }: WidgetProps) => {
   const { enumOptions, enumDisabled } = options;
 
-  const _onChange = ({}, value: any) =>
+  const _onChange = ({
+    target: { value },
+  }: React.ChangeEvent<HTMLInputElement>) =>
     onChange(schema.type == "boolean" ? value !== "false" : value);
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, value);
   const _onFocus = ({
     target: { value },
   }: React.FocusEvent<HTMLInputElement>) => onFocus(id, value);
 
-  const row = options ? options.inline : false;
+  const inline = options.inline !== null ? options.inline : false;
 
   return (
-    <>
-      <FormLabel required={required} htmlFor={id}>
-        {label || schema.title}
-      </FormLabel>
-      <RadioGroup
-        value={`${value}`}
-        row={row as boolean}
-        onChange={_onChange}
-        onBlur={_onBlur}
-        onFocus={_onFocus}>
-        {(enumOptions as any).map((option: any, i: number) => {
-          const itemDisabled =
-            enumDisabled && (enumDisabled as any).indexOf(option.value) != -1;
-
-          const radio = (
-            <FormControlLabel
-              control={<Radio color="primary" key={i} />}
-              label={`${option.label}`}
-              value={`${option.value}`}
-              key={i}
-              disabled={disabled || itemDisabled || readonly}
-            />
-          );
-
-          return radio;
-        })}
-      </RadioGroup>
-    </>
+    <Form.Group controlId={id} className="mb-0">
+      <Form.Label className="d-block">{label || schema.title}</Form.Label>
+      {(enumOptions as any).map((option: any, i: number) => {
+        const itemDisabled =
+          enumDisabled && (enumDisabled as any).indexOf(option.value) != -1;
```

Comment:
This line assumes that the value will either be falsy, or an array (with an `.indexOf` method), but the type signature for WidgetProps.options doesn't guarantee this, so this line can crash at runtime if it's passed any truthy value other than an array.

To be defensive, I'd make sure it's an array:
```suggestion
          Array.isArray(enumDisabled) && enumDisabled.indexOf(option.value) !== -1;
```

Reply:
If we choose to make this change, the same pattern repeats in CheckBoxesWidget and SelectWidget as well, and could be fixed there too

Reply:
@jevakallio Thanks, I will keep this in mind for the future components.